providers/amazon/tests/system/amazon/aws/utils/__init__.py

```diff
@@ -279,6 +280,9 @@ def set_env_id() -> str:
 
 
 def all_tasks_passed(ti) -> bool:
+    if isinstance(ti, RuntimeTaskInstance):
```

Comment:
Nit: perhaps a comment explaining the two branches? (Basically what you have in the PR description)

Reply:
It never hurts 👌